### PR TITLE
[BUGFIX] repaired product add

### PR DESCRIPTION
**Pull request by xandrej**
__Tasks:__
- [x] done
- [ ] todo
- [ ] got feedback

__Informations: cannot add new product

| Q            | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Tests pass?  | yes
| Fixed tickets | https://github.com/massiveart/POOL-ALPIN/issues/1281
| Doc          | none

### CODE REVIEW

**Comment by christianbader** [comment]
add CHANGELOG

File: Resources/public/js/components/products/components/detail-form/main.js [hunk 1]
```diff
@@ -140,11 +140,15 @@ define([
         },
 
         setFormData: function (data) {
-            this.sandbox.form.setData(formSelector, data.toJSON()).then(function () {
+            if (data) {
+                this.sandbox.form.setData(formSelector, data.toJSON()).then(function () {
```
Comment (turbo-ele) [inline Resources/public/js/components/products/components/detail-form/main.js]: remove space between function and ()
````
function() {
```

File: Resources/public/js/components/products/components/detail-form/main.js [hunk 2]
```diff
@@ -140,11 +140,15 @@ define([
...
+                    this.sandbox.start(formSelector);
+                }.bind(this)).fail(function (error) {
```
Comment (turbo-ele) [inline Resources/public/js/components/products/components/detail-form/main.js]: function()

**Comment by christianbader** [comment]
else ack

**Pull request merged**